### PR TITLE
feat(portal): search for mobile mode

### PR DESCRIPTION
**Pull request by mneige**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2-quebec/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The search tool is not adapted to mobile devices

**What is the new behavior?**
In mobile mode (screen size < 768px)  the search bar is at the bottom of the screen for better thumb interactions. The results open in an expansion panel which can open and close by a click or by triggering/clearing the search. The panel will also be used in the future to display the features getInfo. 

TODO : 
- Add "more results" functionality.
- Raising the scalebar to the top left of the map in mobile mode
- Display the footer information in a new map button
- Add a mapOffset

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications:

**Other information**:
image

### CODE REVIEW

File: src/app/pages/footer/footer.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <div class="footer-right" id="footer-id">
+    <div id="scaleLine"></div>
```
Comment (mneige) [inline src/app/pages/footer/footer.component.html:2]: To insert the scalebar

File: src/app/pages/footer/footer.component.scss [hunk 2]
```diff
@@ -38,10 +38,12 @@ span.footer-text a{
 
 ::ng-deep div.ol-scale-line {
     position: fixed !important;
+    background: unset;
     display: flex;
     flex-direction: row;
-    left: 70px !important;
+    left: 2px !important;
```
Comment (mneige) [inline src/app/pages/footer/footer.component.scss:44]: Adjustment related to scalebar new target

File: src/app/pages/menu/menu-pages/about/about.component.html [hunk 3]
```diff
@@ -1,18 +1,13 @@
-<div class="container-fluid page">
-    <div class="container">
-        <div class="row">
+<div class="page">
+    <div class="row">
             <h1 class="d-block themeTitre">En savoir plus</h1>
-            <h2>Titre</h2>
+            <h2>Notre Mission</h2>
 
+            <h3>Mission 1</h3>
```
Comment (mneige) [inline src/app/pages/menu/menu-pages/about/about.component.html:6]: Template

File: src/app/pages/menu/menu.component.scss [hunk 4]
```diff
@@ -28,7 +28,7 @@
     text-decoration: none;
     letter-spacing: 0;
     font-size: 1.1rem;
-    padding: 0 0.5rem 0.6rem;
+    padding: 0 0.5rem 0rem!important;
```
Comment (mneige) [inline src/app/pages/menu/menu.component.scss:31]: adjust mobile navigation arrows height

File: src/qcca-theme/qcca-theme.scss [hunk 5]
```diff
@@ -164,6 +164,10 @@ display: none;
 align-self: flex-end !important;
 }
 
+ul, ol {
```
Comment (mneige) [inline src/qcca-theme/qcca-theme.scss:167]: For the no-result text

File: src/contexts/_default.json [hunk 6]
```diff
@@ -6,7 +6,7 @@
       "title": "Régions administratives",
       "sourceOptions": {
         "type": "wms",
-        "url": "https://geoegl.msp.gouv.qc.ca/apis/wss/amenagement.fcgi",
+        "url": "https://geoegl.msp.gouv.qc.ca/apis/ws/amenagement.fcgi",
```
Comment (mneige) [inline src/contexts/_default.json:9]: fixed typo

File: src/config/config.json [hunk 7]
```diff
@@ -26,13 +26,6 @@
       "available": false
     }
   },
-  "icherche": {
```
Comment (mneige) [inline src/config/config.json:29]: useless setting, it doesn't existe yet

File: src/assets/mapboxStyleExample-vectortile.json [hunk 8]
```diff
@@ -1,92 +1,92 @@
...
-      "layout": {
-        "visibility": "visible",
-        "line-cap": "round",
-        "line-join": "round"
-       }
-    }
-  ],
-  "id": "vectortileid"
-}
+{
```
Comment (mneige) [inline src/assets/mapboxStyleExample-vectortile.json:1]: re-added removed file

File: src/contexts/_default.json (hunk 6 above)
Comment (PhilippeLafreniere18) [reply src/contexts/_default.json:9]: C'est vraiment wss, ws est l'ancienne structure ;)

File: src/contexts/_default.json [hunk 9]
```diff
@@ -20,7 +20,7 @@
       "title": "Aires fauniques communautaires (AFC)",
       "sourceOptions": {
         "type": "wms",
-        "url": "https://geoegl.msp.gouv.qc.ca/apis/wss/environnement.fcgi",
+        "url": "https://geoegl.msp.gouv.qc.ca/apis/ws/environnement.fcgi",
```
Comment (PhilippeLafreniere18) [inline src/contexts/_default.json:23]: C'est vraiment wss, ws est l'ancienne structure ;)

File: src/app/pages/portal/portal.component.ts [hunk 10]
```diff
@@ -427,6 +426,10 @@ export class PortalComponent implements OnInit, OnDestroy {
       ).subscribe(() => {
         this.computeToastPanelOffsetX();
       });
+    // SEARCH MOBILE
+    if (window.screen.availWidth <= 768) { // 768px portrait
```
Comment (PhilippeLafreniere18) [inline src/app/pages/portal/portal.component.ts:430]: Ça vient de ou ce critère ?

Comment (mneige) [reply src/app/pages/portal/portal.component.ts:430]: Du protocole de design du gouvernement

Comment (PhilippeLafreniere18) [inline src/app/pages/portal/portal.component.ts:430]: Il faudrait que ça s'update en temps réel. Présentement, si je change de dimension, le comportement ne suit pas

Comment (mneige) [reply src/app/pages/portal/portal.component.ts:430]: En effet il faut rafraîchir la page. Je vais y travailler

**Review by PhilippeLafreniere18 — APPROVED** [review]

**Pull request merged**